Assets/HoloToolkit/Input/Scripts/InteractableViaEvents.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Use this class for easy access to OnTap, OnGazeEnder, and OnGazeExit events.
+    /// Be sure to override each of these methods with your own implimentation.
+    /// Examples of code usage in each method below in comments.
+    /// </summary>
+    [RequireComponent(typeof(Collider))]
+    public class InteractableViaEvents : MonoBehaviour
```

Comment:
No, this would be a base class that others can inherit from and override the methods as needed.